coverage/lcov-report/postit/server/routes/index.js.html

```diff
@@ -0,0 +1,167 @@
+<span class="cline-any cline-yes">1×</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-yes">1×</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-yes">1×</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-yes">1×</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-yes">1×</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-yes">1×</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-yes">1×</span>
+<span class="cline-any cline-neutral">&nbsp;</span>
+<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">const user = require('../controllers/userController');
+const group = require('../controllers/groupController');
+const usergroup = require('../controllers/usergroupController');
+const message = require('../controllers/messageController');
+const authenticate = require('../middleware/authenticate');
+&nbsp;
+module.exports = (app) =&gt; {
+  app.get('/', (req, res) =&gt; res.status(200).send({
+    message: 'Welcome to PostIt Application, Conversation just became easy',
+  }));
+&nbsp;
+  app.get('/api', (req, res) =&gt; res.status(200).send({
+    message: 'Welcome to PostIt Application API',
+  }));
+&nbsp;
+  // Route to allow users signup
+  app.post('/api/user/signup', user.signUp);
+&nbsp;
+  // Route to allow users signin
+  app.post('/api/user/signin', user.signIn);
+&nbsp;
+  app.use(authenticate.authentication);
+  // Route to allow user create a broadcast group
+  app.post('/api/group', group.create);
+&nbsp;
+  // Route to allow users add other users to group
+  app.post('/api/group/:group_id/user', usergroup.create);
+&nbsp;
+  // Route to allow loggedIn users post messages to created groups
+  app.post('/api/group/:group_id/message', message.create);
+&nbsp;
+  // Route that allows loggedIn users retrieve messages that have been posted to groups
+  app.get('/api/group/:group_id/messages', message.retrieve);
+};
+&nbsp;</pre></td></tr>
+</table></pre>
+<div class='push'></div><!-- for sticky footer -->
+</div><!-- /wrapper -->
+<div class='footer quiet pad2 space-top1 center small'>
+  Code coverage
+  generated by <a href="http://istanbul-js.org/" target="_blank">istanbul</a> at Sat Aug 05 2017 11:49:31 GMT+0100 (W. Central Africa Standard Time)
+</div>
+</div>
+<script src="../../../prettify.js"></script>
+<script>
+window.onload = function () {
+        if (typeof prettyPrint === 'function') {
+            prettyPrint();
+        }
+};
+</script>
+<script src="../../../sorter.js"></script>
+</body>
+</html>
```

Comment:
Please gitignore the entire coverage folder. It should not be committed to version control